src/dbos-executor.ts

```diff
@@ -1179,6 +1172,10 @@ export class DBOSExecutor implements DBOSExecutorContext {
           txn_snapshot = await DBOSExecutor.#retrieveSnapshot(func);
         }
 
+        if (this.debugMode) {
```

Comment:
For stored procs, should we also return the recorded output instead of throwing an error here?